math/dec_test.go

```diff
@@ -671,7 +671,7 @@ func BenchmarkLegacyQuoRoundupMut(b *testing.B) {
 func TestFormatDec(t *testing.T) {
 	type decimalTest []string
 	var testcases []decimalTest
-	raw, err := os.ReadFile("../tx/textual/internal/testdata/decimals.json")
+	raw, err := os.ReadFile("../x/tx/textual/internal/testdata/decimals.json")
```

Comment:
Unrelated to this PR, but I think it does not need another PR to be fixed.
Noticed it was failing while running `make test`: due to https://github.com/cosmos/cosmos-sdk/pull/14634